floris/tools/parallel_computing_interface.py

```diff
@@ -483,10 +445,14 @@ def get_farm_AEP(
         # Evaluate the conditions in floris
         if np.any(conditions_to_evaluate):
             wind_speeds_subset = wind_speeds[conditions_to_evaluate]
+            wind_direction_subset = wind_directions[conditions_to_evaluate]
             yaw_angles_subset = None
             if yaw_angles is not None:
                 yaw_angles_subset = yaw_angles[:, conditions_to_evaluate]
-            self.fi.reinitialize(wind_speeds=wind_speeds_subset)
+            self.fi.reinitialize(
+                wind_directions=wind_direction_subset,
+                wind_speeds=wind_speeds_subset
```

Comment:
I think this is another place where the turbulence_intensities subset should get passed